### PR TITLE
Add wrappers for role, rolebindings, service accounts

### PR DESCRIPTION
**Pull request by ANeumann82**
Check lastPlanUID when checking plan status
Log nested plan status messages

### CODE REVIEW

File: pkg/kudo/instance.go [hunk 1]
```diff
@@ -61,13 +64,44 @@ func ListInstances(client client.Client, namespace string) ([]Instance, error) {
...
 		// We continue, assuming that the plan name is valid and present in OperatorVersion.
 		return kudov1beta1.ExecutionNeverRun, ""
 	}
 
+	ps := instance.Status.PlanStatus[plan]
+	if ps.Status != kudov1beta1.ExecutionFatalError || ps.Message != "" {
+		return ps.Status, ps.Message
+	}
+
+	for _, phaseStatus := range ps.Phases {
```
Comment (nfnt) [inline pkg/kudo/instance.go:93]: Please add a comment why this is necessary. I'd expect that `PlanStatus[plan].Status` is always the source of truth. This indicates that there are conditions where that isn't the case. Is this a bug in KUDO?

File: pkg/kudo/instance.go [hunk 2]
```diff
@@ -61,13 +64,44 @@ func ListInstances(client client.Client, namespace string) ([]Instance, error) {
 	return instances, nil
 }
 
+func currentPlanUID(instance Instance, plan string) apimachinerytypes.UID {
```
Comment (nfnt) [inline pkg/kudo/instance.go]: Nit, isn't the the current `PlanStatus` UID?

**Review by nfnt — APPROVED** [review]
That's a great addition, thanks for adding more types!
Got some smaller comments around naming and clarification around the additional plan status code.

File: pkg/kudo/instance.go (hunk 1 above)
Comment (ANeumann82) [reply pkg/kudo/instance.go:93]: Not sure if it's a bug or a feature, but in a lot of cases the full message is only available in the deepest status item:
https://github.com/kudobuilder/kudo/blob/b1ef318016ad85eaa46bcf00ad23d234a524d609/pkg/engine/workflow/engine.go#L155

Comment (ANeumann82) [reply pkg/kudo/instance.go:93]: It kind of makes sense if you just look at the console output: You don't really want to have the message repeated two or three times

Comment (nfnt) [reply pkg/kudo/instance.go:93]: I see, it's mostly about the message, makes sense.

**Pull request merged**